packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.18.1
+
+* Updates internal API wrapper to make all Flutter methods with a non-null return value be required
```

Comment:
I would mention that this fixes a crash, since that's the client-visible effect of the change.